roles/openshift_default_storage_class/defaults/main.yml

```diff
@@ -15,8 +15,8 @@ openshift_storageclass_defaults:
     provisioner: azure-disk
     name: azure-standard
     parameters:
-      storageaccounttype: 'Standard_LRS'
-      kind: 'Shared'
+      storageaccounttype: "{{ openshift_cloudproviders_azure_storage_type | default ('Standard_LRS') }}"
```

Comment:
I'd move this defaulting to roles/openshift_default_storage_class/defaults/main.yml in this role and we need to make sure to add any new configuration variables to inventory/hosts.example, we probably need an Azure section if we don't already have one.